components/dashboard/src/projects/Project.tsx

```diff
@@ -80,36 +78,13 @@ export default function () {
         entries.push({
             title: "Trigger Prebuild",
```

Comment:
**question**: What do you think of renaming the action verb to _Run_ and change this to _Run Prebuild_ to make it sound less complex? We can keep the metadata information with the same verb (Triggered 2 minutes ago). Maybe also something to change in a future iteration.

Reply:
Ah, I was confused by what this "Trigger Prebuild" meant.

What is the intent? Is it to "Re-try Failed Prebuild"? This sounds like a good stretch goal, that could be removed for now. 😇

Otherwise, I can't imagine what "Triggering" an already existing & successful prebuild even means.

Reply:
Also, the "New Workspace" context menu entry is a duplicate of the "New Workspace" button that appears on hover, right? Probably only keep one of these two.

Reply:
@gtsiolis the discussion about triggering prebuilds seems to be too important to get lost in the comments here. should we tackle it separately?

Reply:
@AlexTugarev Absolutely! Added https://github.com/gitpod-io/gitpod/issues/5008 to continue the discussion for this. 🙏 

🍒 🍒 🍒 🍒 

No strong feelings about the duplicate _New Workspace_ context menu entry. I think it will become more clear later whether we want to hide options in the context menu or surface them on each entry. I'd vote for moving forward with a boring (simple) solution of removing the _New Workspace_ entry from the dropdown. 🏓

Reply:
```suggestion
            title: "Rerun Prebuild",
```